packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.vue

```diff
@@ -12,18 +12,24 @@
       @blur="$emit('blur')"
     />
     <!-- @slot -->
-    <slot name="icon">
-      <span v-if="icon" class="sf-search-bar__icon">
-        <SfIcon :color="icon.color" :size="icon.size" icon="search" />
-      </span>
-    </slot>
+    <SfButton
+      class="sf-search-bar__button sf-button--pure"
+      @click="$emit('click', $event.target.value)"
```

Comment:
R: probably target of click is button or I missing something ;)

Reply:
You are right, I fixed that.